res/css/structures/_SpacePanel.pcss

```diff
@@ -277,14 +277,11 @@ $activeBorderColor: $primary-content;
         .mx_BaseAvatar:not(.mx_UserMenu_userAvatar_BaseAvatar) .mx_BaseAvatar_initial {
             color: $secondary-content;
             border-radius: 8px;
-            background-color: $panel-actions;
-            font-size: $font-15px !important; /* override inline style */
             font-weight: $font-semi-bold;
             line-height: $font-18px;
-
-            & + .mx_BaseAvatar_image {
-                visibility: hidden;
-            }
+            /* override inline styles which are part of the default avatar style as these uses a monochrome style */
```

Comment:
it feels like we shouldn't be using inline styles, *especially* if they they have to be overridden in the css file. but I guess that's an existing situation which isn't getting materially worse here.

Reply:
We can't avoid using inline styles for things where the colours/sizes are defined in Javascript. The space panel is the one place which doesn't use coloured default avatars, we could override them in JS but this seems cleaner.

Reply:
Hrm, I guess. Does the font size also need to be defined in javascript?

(Also, I wonder if we should have a comment at the point that the styles are set in javascript that they will be overridden for the space panel)

Reply:
not sure about the font size, that one is quite old